### PR TITLE
Fix default metrics_resolution language

### PR DESCRIPTION
**Pull request by lpsantil**
PR for https://github.com/openshift/openshift-docs/issues/4412

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@lpsantil Thank you! These changes look good to me. Can you please squash this into one commit, however? That will help us with seamless cherry-picking and publishing. Thanks again!

**Comment by ahardin-rh** [comment]
@lpsantil Actually, I  just noticed that this PR is submitted against enterprise-3.5. Can you please submit it against the master branch, then we can cherry-pick it to the other relevant branches like 3.5? Thanks! Sorry for any confusion!

**Comment by lpsantil** [comment]
@ahardin-rh @bfallonf Would you mind reviewing https://github.com/openshift/openshift-docs/issues/4411 before I push this stuff to master?

**Comment by bfallonf** [comment]
@lpsantil I'm trying to figure out the deal with that PR right now. I think different versions having different capacities might impact what happens there.

That said, I think this PR can merge without affecting that one, so feel free to move ahead with this.

**Comment by ahardin-rh** [comment]
Looks like this is still not resolved. I think we should check with @jeremyeder if this guidance is correct for OCP 3.5 and, if so, if this guidance should be updated for other OCP versions as well. If not, we may not need to submit this against master.

**Comment by vikram-redhat** [comment]
@ahardin-rh @lpsantil - I have replied in #4411 .

File: install_config/cluster_metrics.adoc [hunk 1]
```diff
@@ -185,8 +185,8 @@ These two test cases are presented on the following graph:
 image::https://raw.githubusercontent.com/ekuric/openshift/master/metrics/1_10kpods.png[1000 pods vs 10000 pods monitored during 24 hours]
 endif::openshift-origin[]
 
-If the default value of 7 days for `openshift_metrics_duration` and 10 seconds for
-`openshift_metrics_resolution` are preserved, then weekly storage requirements for the Cassandra pod would be:
+If the default value of 7 days for `openshift_metrics_duration` is preserved, and
```
Comment (kalexand-rh) [inline install_config/cluster_metrics.adoc]: s/preserved,/preserved

File: install_config/cluster_metrics.adoc [hunk 2]
```diff
@@ -185,8 +185,8 @@ These two test cases are presented on the following graph:
...
+`openshift_metrics_resolution` is set to 10 seconds, then weekly storage requirements for the Cassandra pod would be:
```
Comment (kalexand-rh) [inline install_config/cluster_metrics.adoc]: s/would be/are

**Review by kalexand-rh — COMMENTED** [review]
@ahardin-rh, is this PR still valid?

**Pull request closed without merging**